src/icon/mixins.scss

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import './variables';
+@import '../style/index';
```

Comment:
Well then need to go through all the imports of `/variables` and remove/replace them.

Reply:
This is now done. Should be good to merge